fuzztests/runFuzzTests.js

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/node
+
+// Helper script that searches for Jest fuzz tests in the current directory and
+// executes them im new processes using the found fuzz test names.
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const fs = require("fs/promises");
+const { spawn } = require("child_process");
+
+const fuzzTestFileExtension = "fuzz.js";
+const fuzzTestNameRegex = /it.fuzz\("(.*)"/g;
+
+async function findFuzzTestNamesInFile(file) {
+	const fuzzTestNames = [];
+	if (file.endsWith(fuzzTestFileExtension)) {
+		const content = await fs.readFile(file);
+		for (let match of content.toString().matchAll(fuzzTestNameRegex)) {
+			fuzzTestNames.push(match[1]);
+		}
+	}
+	return fuzzTestNames;
+}
+
+async function findFuzzTestNamesInDir(dir) {
+	const files = await fs.readdir(dir);
+	let fuzzTests = {};
+	for (const file of files) {
+		const fuzzTestNames = await findFuzzTestNamesInFile(file);
+		if (fuzzTestNames.length) {
+			fuzzTests[file] = fuzzTestNames;
+		}
+	}
+	return fuzzTests;
+}
+
+async function executeFuzzTest(file, name) {
+	console.log(`--- Executing fuzz test ${file} > ${name}`);
+	return new Promise((resolve, reject) => {
+		process.env["JAZZER_FUZZ"] = "1";
+		let test = spawn("npm", ["test"], { stdio: "inherit" });
```

Comment:
Yes, that's just wrong! :grimacing: 
The intention was to execute one fuzz test at a time using `--testNamePattern`. The assumption is that the test names are distinct enough to execute only one specific fuzz test, so that we don't need more elaborate parsing.  Changed the call accordingly now.